### PR TITLE
upgrade to 1.5.10

### PR DESCRIPTION
**Pull request by Jibec**
Fix for https://github.com/YunoHost-Apps/grav_ynh/issues/20
Upgrade to 1.5.8
Fix minor packaging issue php-fpm.ini removal and add a type for the language field of manifest
Fix shellcheck warnings

### CODE REVIEW

File: conf/app.src [hunk 1]
```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/getgrav/grav/releases/download/1.5.3/grav-admin-v1.5.3.zip
-SOURCE_SUM=055F6F248AA74C68436B6D639D9340D333136044E5946FC6D62271AD2B3B319B
+SOURCE_URL=https://github.com/getgrav/grav/releases/download/1.5.8/grav-admin-v1.5.8.zip
+SOURCE_SUM=3b0210bf8639c1d75b046e4a9b364b216b41c449c8b5ac74fd71c098d25cb9bb
```
Comment (decentral1se) [inline conf/app.src]: The sums don't matct? It's different to the one in app-upgrade.src?

Not sure if that is meant to be the case ;)

File: scripts/upgrade [hunk 2]
```diff
@@ -15,36 +15,36 @@ source /usr/share/yunohost/helpers
...
 #=================================================
 
 # Fix is_public as a boolean value
 if [ "$is_public" = "Yes" ]; then
-	ynh_app_setting_set $app is_public 1
+	ynh_app_setting_set "$app" is_public 1
 	is_public=1
 elif [ "$is_public" = "No" ]; then
-	ynh_app_setting_set $app is_public 0
+	ynh_app_setting_set "$app" is_public 0
```
Comment (decentral1se) [inline scripts/upgrade]: Some of the high quality apps don't quote like this in their install scripts.

See https://github.com/YunoHost-Apps/gitlab_ynh/blob/testing/scripts/install#L80.

I'm not sure but I would do what they do ...

File: scripts/restore [hunk 3]
```diff
@@ -27,17 +27,17 @@ ynh_abort_if_errors
 
 app=$YNH_APP_INSTANCE_NAME
 
-domain=$(ynh_app_setting_get $app domain)
-path_url=$(ynh_app_setting_get $app path)
-final_path=$(ynh_app_setting_get $app final_path)
+domain=$(ynh_app_setting_get "$app" domain)
```
Comment (tituspijean) [inline scripts/restore]: Command style for retrieving the settings is now `domain=$(ynh_app_setting_get --app=$app --key=domain)
`. Should we refactor in another PR, or wait and do it in this one?

File: scripts/upgrade (hunk 2 above)
Comment (tituspijean) [reply scripts/upgrade]: I have no opinion here. It looks like a quotation-mark-them-all frenzy, but with little side effects?

Comment (Jibec) [reply scripts/upgrade]: https://www.shellcheck.net recommands it, and I never noticed any impact.

File: scripts/restore (hunk 3 above)
Comment (Jibec) [reply scripts/restore]: in a new PR, but that's not bringing any value in the app..

**Pull request merged**